src/server/modes/charts/plugins/ql/utils/visualization-utils.ts

```diff
@@ -191,8 +193,10 @@ const mapItems = ({items, fields}: {items: Field[]; fields: Field[]}) => {
             }
         }
 
-        // TODO: else we need to make this field invalid
-        return item;
+        return {
+            ...item,
+            conflict: 'not-existing-ql',
```

Comment:
if there is some kind of enum or constant, it is better to use it instead of a string